src/docs/upgrading.dox

```diff
@@ -1,14 +1,53 @@
 /*! @page upgrading Upgrading WiredTiger applications
 
 </dl><hr>
-@section version_322 Upgrading to Version 3.2.2
+@section version_1000 Upgrading to Version 10.0.0
 <dl>
 
+<dt>LAS and HS files</dt>
+<dd>
+The WiredTigerLAS.wt file will no longer be generated by the cache overflow mechanism.
+Instead, a WiredTigerHS.wt file will be generated as a history store for updates.
+Same as other files generated and maintained by WiredTiger storage engine, no manual
+intervention should be performed to the history store file.
+</dd>
+
+<dt>Default transaction isolation level</dt>
+<dd>
+The default transaction isolation level has been changed from "read-committed" to "snapshot"
+in WiredTiger.
+</dd>
```

Comment:
@kommiharibabu  Have we also restricted read-committed and read-uncommitted transactions to be read-only? If we haven't done that yet, I'd like to get it into the same release.

Reply:
@agorrod We didn't yet convert the read-committed and read-uncommitted transactions to read-only. I didn't yet find the reason behind the performance regression with WT-6710, I fee either I can revert and analyze the regression.

Reply:
@lukech WT-6710 is reverted. Can we remove updates from WT-6710.

Reply:
@kommiharibabu I'm happy to wait for WT-6710 to be merged again, and changes for the read-only read-(un)committed transactions to land. They are critical changes for our standalone release users.

Reply:
@lukech Both WT-6710 (default isolation to snapshot) and WT-6677 (read-committed and read-uncommitted isolation to read-only operations) tickets are merged now. 

we can include the WT-6677 in the description along with WT-6710 and you can finalize the release once it is stable.

Reply:
Thanks @kommiharibabu. I merged in develop branch, and pushed 913d1b1 to add WT-6677 into NEWS and upgrading doc.